src/system/resources/service/control.py

```diff
@@ -29,5 +29,10 @@ def post(cls):
     def validate_and_create_service_cmd(cls, action: str, service: str) -> str:
         if service in Services.__members__.keys():
             service_file_name = Services[service].value.get('service_file_name')
-            return "sudo systemctl {} {}".format(action, service_file_name)
+            cmd: str = ""
+            if action == "start":
+                cmd = f"sudo systemctl enable {service_file_name} &&"
+            elif action == "stop":
+                cmd = f"sudo systemctl disable {service_file_name} &&"
+            return f"{cmd} sudo systemctl {action} {service_file_name}".strip()
```

Comment:
Make it DRY, just make a function on `def create_service_cmd(action, service_file_name)` & call it from both location.